packages/core/src/lib/icon/icons.ts

```diff
@@ -1,10 +1,19 @@
 import * as MDI from '@mdi/js';
 
+type Path = string;
+
+export interface CustomIcon {
+  path: string;
+  opacity?: number | undefined;
+}
```

Comment:
Alternatively to re-exporting the icon, you could try something to incorporate different viewboxes into the path data interfaces. I think that makes everything a bit more complicated when we could simply use a standard icon size, so I wouldn't necessarily recommend it